qiskit/ignis/experiments/calibration/instruction_data/database.py

```diff
@@ -61,7 +67,9 @@ def get_dataframe(
         Args:
             qubits: Index of qubit(s) to search for.
             channel: Label of pulse channel to search for. Wildcards can be accepted.
-            gate_type: Name of gate to search for. Wildcards can be accepted.
+            inst_name: Name of gate to search for. Wildcards can be accepted.
+            pulse_type: Name of ParametricPulse generator that is used for pulse creation.
+            stretch: Stretch factor of the pulse.
```

Comment:
```suggestion
            stretch_factor: Stretch factor of the pulse, typically used for error mitigation.
```
Can you change `stretch` to `stretch_factor` in the rest of the code?